www/package.json

```diff
@@ -8,6 +8,6 @@
   "license": "MIT",
   "dependencies": {
     "@evergreen-wc/eve-button": "^0.0.9",
-    "@evergreen-wc/eve-container": "^0.0.9"
+    "@evergreen-wc/eve-container": "^0.0.10"
```

Comment:
I think I would still be good to keep #366 open and have that handle just to isolate the dependency related changes from the new enhancement work.

Reply:
Actually, this is probably small enough and closely related enough that it should be fine.  I had to do something similar for #367 and so probably makes sense and takes the most advantage of a monorepo.  Up to you.